### PR TITLE
docs: wallet endpoint

### PR DESCRIPTION
**Pull request by tsanta-niaina-MT**

### CODE REVIEW

**Review by sanda03 — COMMENTED** [review]
you forgot the table "category"

File: doc/api.yml [hunk 1]
```diff
@@ -51,8 +51,354 @@ paths:
           $ref: '#/components/responses/TooManyRequests'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /Account:
```
Comment (RickaPrincy) [inline doc/api.yml]: all path must be in lowercase 
ex:  this is wrong  /Account/{AccountId})

File: doc/api.yml [hunk 2]
```diff
@@ -51,8 +51,354 @@ paths:
...
+          $ref: '#/components/responses/NotFound'
+        '429':
+          $ref: '#/components/responses/TooManyRequests'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+    put:
+      tags:
+        - Account
+      summary: Create or Update Account
+      operationId: crupdateAccount
```
Comment (RickaPrincy) [inline doc/api.yml:81]: Make all crupdate requestBody an Array

File: doc/api.yml [hunk 3]
```diff
@@ -51,8 +51,354 @@ paths:
...
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '429':
+          $ref: '#/components/responses/TooManyRequests'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+  /Transaction:
```
Comment (RickaPrincy) [inline doc/api.yml]: transactions
Make them in plural 

/accounts
/transactions
...

File: doc/api.yml [hunk 4]
```diff
@@ -51,8 +51,354 @@ paths:
...
+          $ref: '#/components/responses/NotFound'
+        '429':
+          $ref: '#/components/responses/TooManyRequests'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+    put:
+      tags:
+        - Transaction
+      summary: create or update Transaction
+      operationId: CreateOrUpdateTransaction
```
Comment (RickaPrincy) [inline doc/api.yml:180]: missing requestBody

File: doc/api.yml [hunk 5]
```diff
@@ -51,8 +51,354 @@ paths:
...
+          $ref: '#/components/responses/NotFound'
+        '429':
+          $ref: '#/components/responses/TooManyRequests'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+    put:
+      tags:
+        - Transfert
+      summary: create or update Transfert
+      operationId: CreateOrUpdateTransfert
```
Comment (RickaPrincy) [inline doc/api.yml:232]: missing request body

File: doc/api.yml [hunk 6]
```diff
@@ -51,8 +51,354 @@ paths:
...
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '429':
+          $ref: '#/components/responses/TooManyRequests'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+  /balance_history:
```
Comment (RickaPrincy) [inline doc/api.yml]: balances

**Review by RickaPrincy — CHANGES_REQUESTED** [review]
All put or post should have a request body  (make the request body an array)

**Pull request merged**